PackageExplorer/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Fody" version="1.29.4" targetFramework="net45" developmentDependency="true" />
```

Comment:
[Fody](https://github.com/Fody/Fody) is a tool that makes certain things easier :smile: when it comes to getting right of "boiler plate" code.  I use it to handle the code for [PropertChanged](https://github.com/Fody/PropertyChanged) notifications a lot as the code for that is almsot 100% redundant in all cases.